ownCloudAppShared/AppLock/PasscodeViewController.swift

```diff
@@ -285,6 +285,14 @@ public class PasscodeViewController: UIViewController, Themeable {
 
 	// MARK: - Themeing
 	public override var preferredStatusBarStyle : UIStatusBarStyle {
+		if VendorServices.shared.isBranded {
+			if #available(iOSApplicationExtension 13.0, *) {
+				return .darkContent
+			} else {
+				return .default
+			}
```

Comment:
Are these styles really applicable to all brandings? As-is, overriding these isn't possible.

Reply:
I figured out, that the general status bar style does not work for branded apps and can be static for this view. At the moment it cannot be overridden, if this is needed, we can add another parameter to the `ThemeCollection` class.